### DISCUSSION TITLE
Exclude not working

### DISCUSSION THREAD

**Discussion by Inaki74** [JetBrains/Qodana#244] [Q&A]
### Discussed in https://github.com/JetBrains/Qodana/discussions/109

<div type='discussions-op-text'>

<sup>Originally posted by **burberius** July 29, 2022</sup>
I try to scan a project on GitLab CI with Qodana and specified the Qodana cache into the project folder so it can be cached by GitLab CI.
But Qodana also scans the Gradle in there and finds some hundret problems.

So I tried to exclude it with, but that is not recognized! I tried relative paths, absolute and also tried to exclude some of the project files, but it was all ignored!

```
version: "1.0"
profile:
  name: qodana.recommended
exclude:
  - name: ALL
    paths:
      - qodana/cache
```

Can it be broken in the latest release? Unfortunatelly I couldn't verify it by using an old one...</div>

Hi, I've been having this problem so I decided to revive this issue.

This is my exclude in my qodana.yaml:
![image](https://github.com/JetBrains/Qodana/assets/42726328/aa95c8b1-6c65-45af-83cd-62f1d299538b)

Which should be working, however when I see the actual files being analyzed I see this:
![image](https://github.com/JetBrains/Qodana/assets/42726328/1f14e6d5-f403-4107-b955-690e3f9cf13d)

Getting the GameAssembly is ok since its within the Assets/Project/Code folder, the one I'm interested in.
However Its getting the contents of the Library folder, which I told Qodana to ignore specifically.

Am I doing something wrong here? Thanks for the help in advance.

**Comment by Inaki74** [comment]
Ah, in case it is needed, here is my whole qodana.yaml file as an image because for some reason its not showing as code:

![image](https://github.com/JetBrains/Qodana/assets/42726328/ca47ebea-5604-4aa9-a08e-3538200ce63e)

**Comment by Inaki74** [comment]
Bump

**Comment by tiulpin** [comment]
>So I tried to exclude it with, but that is not recognized! I tried relative paths, absolute and also tried to exclude some of the project files, but it was all ignored!

Hello! Could you please also share your GitLab configuration? Our default example there is with `.qodana` directory, not `qodana`, maybe that's why the exclude is not working as expected
